packages/ui-components/src/components/Button.tsx

```diff
@@ -208,19 +205,19 @@ export const GreyOutlinedButton = styled(OutlinedButton)`
   }
 `;
 
-const StyledLink = styled(MuiLink)`
+const StyledLink = styled(MuiLink)<OverrideableComponentProps<MuiLinkProps>>`
```

Comment:
Does OverrideableComponentProps need to be imported?

Reply:
So it is, but this is one of those ones where it might have been lost in translation with my PR splitting - potentially is done elsewhere/will be a part of the overall branch that we can do a PR for if not?

Reply:
Yes i can see it in the other pr